common/src/main/java/org/apache/celeborn/common/meta/FileManagedBuffers.java

```diff
@@ -24,7 +24,8 @@
 import org.apache.celeborn.common.network.buffer.ManagedBuffer;
 import org.apache.celeborn.common.network.util.TransportConf;
 
-public class FileManagedBuffers {
+public class FileManagedBuffers implements Buffers {
```

Comment:
Making this class able to read shuffle files and shuffle memory cache will make code more readable.